### PR TITLE
fix: value extraction of point cloud colors enum in display

### PR DESCRIPTION
**Pull request by anna-charlotte**
Fix code snippet for point cloud display with color:
Add `PointCloudEnum.COLORS` as name tag instead of `'point_cloud_colors'`

- [x] check and update documentation, if required. See [guide](https://github.com/jina-ai/jina/blob/master/CONTRIBUTING.md#documentation-guidelines)

### CODE REVIEW

File: docarray/document/mixins/mesh.py [hunk 1]
```diff
@@ -101,9 +101,9 @@ def load_vertices_and_faces_to_point_cloud(self: 'T', samples: int) -> 'T':
         faces = None
 
         for chunk in self.chunks:
-            if chunk.tags['name'] == MeshEnum.VERTICES:
+            if chunk.tags['name'] == MeshEnum.VERTICES.value:
```
Comment (samsja) [inline docarray/document/mixins/mesh.py:104]:
```suggestion
            if chunk.tags['name'] in  [MeshEnum.VERTICES.value, MeshEnum.VERTICES]
```
for sake of compatibility with previous version

File: docarray/document/mixins/mesh.py [hunk 2]
```diff
@@ -101,9 +101,9 @@ def load_vertices_and_faces_to_point_cloud(self: 'T', samples: int) -> 'T':
...
                 vertices = chunk.tensor
-            if chunk.tags['name'] == MeshEnum.FACES:
+            if chunk.tags['name'] == MeshEnum.FACES.value:
```
Comment (samsja) [inline docarray/document/mixins/mesh.py:106]:
```suggestion
            if chunk.tags['name'] in  [MeshEnum.VERTICES.value, MeshEnum.VERTICES]
```

same here

File: docarray/document/mixins/plot.py [hunk 3]
```diff
@@ -133,7 +133,10 @@ def _is_3d_vertices_and_faces(self):
         """
         if self.chunks is not None:
             name_tags = [c.tags['name'] for c in self.chunks]
-            if MeshEnum.VERTICES in name_tags and MeshEnum.FACES in name_tags:
+            if (
+                MeshEnum.VERTICES.value in name_tags
+                and MeshEnum.FACES.value in name_tags
+            ):
```
Comment (samsja) [inline docarray/document/mixins/plot.py:139]: same lets support the old way as well

File: docarray/document/mixins/mesh.py (hunk 1 above)
Comment (samsja) [reply docarray/document/mixins/mesh.py:104]: okay it was not released so not needed actually

**Review by samsja — APPROVED** [review]

**Review by JohannesMessner — APPROVED** [review]

**Pull request merged**